pkgs/top-level/all-packages.nix

```diff
@@ -8225,7 +8225,7 @@ with pkgs;
 
   kexec-tools = callPackage ../os-specific/linux/kexec-tools { };
 
-  keepkey_agent = with python3Packages; toPythonApplication keepkey_agent;
```

Comment:
Those should all get aliases, too.

Reply:
Good point, and the python module needed fixing as well:

```
  keepkey-agent = with python3Packages; toPythonApplication keepkey_agent;
  trezor-agent = with python3Packages; toPythonApplication trezor_agent;
  ledger-agent = with python3Packages; toPythonApplication ledger_agent;
```